ubuntu.log

```diff
@@ -0,0 +1,26 @@
+Using default tag: latest
```

Comment:
@tonistiigi Oh I am so sorry about that! Did a `git add -A` and didn't pay enough attention. The PR has been updated now. Thanks for the help!